Description: install kconf_update scripts
 install the kconf update scripts in the right location for kmix which
 is kdelibs 4 
Author: Jonathan Riddell <jr@jriddell.org>
Origin: vendor
Forwarded: https://git.reviewboard.kde.org/r/125706/
Last-Update: 2015-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/data/CMakeLists.txt b/data/CMakeLists.txt
index b652a0f..b7b13e7 100644
--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -1,2 +1,14 @@
+# installed twice for kmix builds on kdelibs4 and kf5
+execute_process(COMMAND kde4-config --install data
+                OUTPUT_VARIABLE KDE4_DATA_DIR
+	        OUTPUT_STRIP_TRAILING_WHITESPACE
+	        RESULT_VARIABLE kde4config_failed)
+
+if (kde4config_failed)
+    message(FAILED "Couldn't find kde4-config from kde4libs for kconf_update files")
+else()
+    install(FILES disable_kmix.upd DESTINATION ${KDE4_DATA_DIR}/kconf_update)
+    install(PROGRAMS plasmaVolumeDisableKMixAutostart.pl DESTINATION ${KDE4_DATA_DIR}/kconf_update)
+endif()
 install(FILES disable_kmix.upd DESTINATION ${CMAKE_INSTALL_PREFIX}/share/kconf_update)
 install(PROGRAMS plasmaVolumeDisableKMixAutostart.pl DESTINATION ${CMAKE_INSTALL_PREFIX}/share/kconf_update)
